src/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface IRecipe {
```

Comment:
same as above https://medium.com/@koss_lebedev/type-aliases-vs-interfaces-in-typescript-based-react-apps-e77c9a1d5fd0